prototype/mobile_endpoint/views/receiver.py

```diff
@@ -0,0 +1,31 @@
+from flask import Blueprint, request
+
+from mobile_endpoint.case.case_processing import process_cases_in_form
+from mobile_endpoint.dao import SQLDao
+from mobile_endpoint.extensions import requires_auth
+from mobile_endpoint.form.form_processing import create_xform, get_instance_and_attachment, get_request_metadata
+from mobile_endpoint.views.response import get_open_rosa_response
+
+
+mod = Blueprint('receiver', __name__, url_prefix='/receiver')
+
+
+@mod.route('/<domain>', methods=['POST'])
+@requires_auth
+def form_receiver(domain):
+    instance, attachments = get_instance_and_attachment(request)
+    request_meta = get_request_metadata(request)
+    request_meta['domain'] = domain
+
+    dao = SQLDao()
+
+    xform_lock = create_xform(instance, attachments, request_meta, dao)
```

Comment:
we should decide whether or not we want to support atomic edits or not (and if so how). would be fine to make that a separate endpoint.